packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1454,7 +1455,30 @@ async function handler(
       metadata: reqBody.metadata,
       attendees: {
         createMany: {
-          data: attendeesData,
+          data: [
+            ...evt.attendees.map((attendee) => {
+              //if attendee is team member, it should fetch their locale not booker's locale
+              //perhaps make email fetch request to see if his locale is stored, else
+              const retObj = {
+                name: attendee.name,
+                email: attendee.email,
+                timeZone: attendee.timeZone,
+                locale: attendee.language.locale,
+              };
+              return retObj;
+            }),
+            // Have this for now until we change the relationship between bookings & team members
```

Comment:
Is this bit needed? also could you add it to the attendeesData? https://github.com/calcom/cal.com/pull/7756/files#diff-a7c21727d9a0d4d1aaff9d24fca9293bc28468861bc8022dfa03116908b6fae7R1432

Reply:
Can we create a helper for this?